packages/nimble-components/src/number-field/specs/IxD.md

```diff
@@ -0,0 +1,146 @@
+    - Step size is typically 1 but can vary (e.g., 5, 10).
+    - Max/min constraints are enforced, and values are coerced to stay within
+      range.
+    - _`nimble-number-field` supports this use case today._
+
+2. **Integer Input (Without Steppers)**
+
+    - Used when users need to input whole numbers without a constrained range.
+    - Validates input to ensure only integers are accepted (e.g., rejects 3.5 or
+      “abc”).
+    - May include min/max constraints (e.g., age between 0 and 150).
+
+3. **Decimal Values**
+    - Used for precise, fractional numbers (e.g., measurements or scientific
+      data).
+    - Defines decimal precision (e.g., 2 places for currency, 3 for scientific
+      data) and coerces values to enforce it.
+    - Does not use increment/decrement buttons.
+    - Supports locale-specific formatting (e.g., 1.23 vs. 1,23).
+
+## Requirements
+
+1. Support locale-specific formatting for decimal values (e.g., 10212,456 in
+   German).
+2. Allow specifying decimal precision (e.g., 2 places → 123.45).
+3. Provide validation-only mode for max/min, supporting both coercion and
+   non-coercion on blur.
+
+### Anatomy
+
+![ ](spec-images/number-field-anatomy.png)
+
+| Element             | Description                                                                                      |
+| ------------------- | ------------------------------------------------------------------------------------------------ |
+| Label               | Name for the field.                                                                              |
+| Required            | Indicates whether the field is required or optional.                                             |
+| Placeholder         | Shows an example of the data required in the field or a description of the purpose of the field. |
+| Numeric value       | Current control value.                                                                           |
+| Increment/Decrement | Changes numeric value by the step size.                                                          |
+| Error status icon   | Appears when a user has entered invalid data in a field.                                         |
+| Error text          | Appears when a user has entered invalid data in a field.                                         |
+
+### Related Components
+
+- [Number Field](https://nimble.ni.dev/storybook/index.html?path=/docs/components-number-field--docs)
+
+## Design
+
+### Configuration
+
+Proposed attributes:
+
+| Attribute               | Description                                                                                                               |
+| ----------------------- | ------------------------------------------------------------------------------------------------------------------------- |
+| coerce-invalid          | Boolean - Whether to coerce invalid max/min values on blur.                                                               |
```

Comment:
I don't think we need an attribute for this.
 - `coerce-invalid=true` is the existing `nimble-number-field` behavior when `min` and `max` are set
 - `coerce-invalid=false` is the existing `nimble-number-field` behavior when `min` and `max` are not set
In the latter case an application can contain logic to set `error-text` and `error-visible` and disable form submission.

Reply:
Sounds good - removed.